File: api/src/main/java/com/example/api/dto/enums/BookingStatus.java
package com.example.api.dto.enums;

/**
 * Статус бронирования.
 */
public enum BookingStatus {

    /**
     * Подтверждено владельцем.
     */
    APPROVED,

    /**
     * Отменено создателем.
     */
    CANCELED,

    /**
     * Отклонено владельцем.
     */
    REJECTED,

    /**
     * Ожидает одобрения.
     */
    WAITING
}

File: api/src/main/kotlin/com/example/api/dto/enums/BookingStatus.kt
package com.example.api.dto.enums

/**
 * Статус бронирования.
 */
enum class BookingStatus {

    /**
     * Подтверждено владельцем.
     */
    APPROVED,

    /**
     * Отменено создателем.
     */
    CANCELED,

    /**
     * Отклонено владельцем.
     */
    REJECTED,

    /**
     * Ожидает одобрения.
     */
    WAITING
}
